src/providers/spatialite/qgsspatialiteprovider.cpp

```diff
@@ -5820,52 +5820,61 @@ bool QgsSpatiaLiteProvider::getTableSummaryAbstractInterface( gaiaVectorLayerPtr
 
 bool QgsSpatiaLiteProvider::getTableSummary()
 {
-  int ret;
-  int i;
-  char **results = nullptr;
-  int rows;
-  int columns;
-  char *errMsg = nullptr;
+  mLayerExtent.setMinimal();
```

Comment:
setMinimal is rather confusing. Could we add "setToNull" instead and deprecate setMinimal?

Reply:
I'm very happy to add a setNull() to QgsRectangle and even to deprecate setMinimal, but is this the right PR ? Or should I just merge this back into PR GH-54646 and handle everything there ?

Reply:
Let's do it in a separate PR

Reply:
I've added setNull() and deprecated setMinimal() in GH-54646

Reply:
Given the deprecation of QgsRectangle::setMinimal() introduced other problems I put it in its own PR here: https://github.com/qgis/QGIS/pull/54934

@nyalldawson if we can merge this in while CI-green that'd be great, then GH-54934, if legit at this stage :)